<launch>
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="usb_cam" />
        <param name="io_method" value="mmap"/>
    </node>

    <node pkg="nodelet"
          type="nodelet"
          name="qr_detector"
          output="screen"
          args="standalone robot_vision/qr_detector_nodelet">
        <remap from="image" to="usb_cam/image_raw"/>
    </node>

    <node
        pkg="rqt_image_view"
        type="rqt_image_view"
        name="rqt_image_view"
        output="screen"
    />    
</launch>
